.clang-format

```diff
@@ -16,4 +16,4 @@ BinPackParameters: false
 AlwaysBreakTemplateDeclarations: Yes
 ReflowComments: false
 BinPackArguments: false
-BinPackParameters: false
+BinPackParameters: false
```

Comment:
this file now has a newline.

Reply:
Quick question, as I'm still learning, aren't missing newlines caught by code checks?

Reply:
Code checks (that run `clang-format` and `clang-tidy`) inspect only C++ code.

Reply:
Ah, got it. Thanks.